cirq-core/cirq/protocols/json_serialization.py

```diff
@@ -256,6 +256,7 @@ def default(self, o):
                 sympy.StrictLessThan,
                 sympy.Equality,
                 sympy.Unequality,
+                sympy.Xor,
```

Comment:
Should we add other boolean operators as well, e.g. `sympy.And`, `sympy.Not`, `sympy.Or`?

Reply:
This needs to go a few other places as well: https://github.com/search?q=repo%3Aquantumlib%2FCirq+strictGreaterThan&type=code